test/Feature/AndOrParamsSpec.hs

```diff
@@ -228,9 +242,10 @@ spec actualPgVersion =
     context "used with DELETE" $
       it "succeeds when using and/or params" $
         request methodDelete "/grandchild_entities?or=(id.eq.1,id.eq.2)&select=id,name"
-          [("Prefer", "return=representation")] "" `shouldRespondWith`
-          [json|[{ "id": 1, "name" : "updated grandchild entity"},{ "id": 2, "name" : "updated grandchild entity"}]|]
```

Comment:
Some tests expected the "updated" state of a row from a previous test, `"updated grandchild entity"` here. Those had to be changed to expect the original, unchanged value.